docs/source/guide/helm_values.md

```diff
@@ -34,166 +34,173 @@ meta_description: For cases when you want to customize your Label Studio Enterpr
+| `app.containerSecurityContext.runAsNonRoot`             | string  | true              | Avoid running as root User.                                                                  |
+| `app.containerSecurityContext.allowPrivilegeEscalation` | string  | false             | Controls whether a process can gain more privileges than its parent process.                 |
+| `app.topologySpreadConstraints`                         | list    | [ ]               | Topology Spread Constraints for pod assignment.                                              |
+| `app.dnsPolicy`                                         | string  | ""                | Pod DNS policy.                                                                              |
+| `app.enableServiceLinks`                                | boolean | false             | Service environment variables.                                                               |
+| `app.shareProcessNamespace`                             | boolean | false             | Enable shared process namespace in a pod.                                                    |
+| `app.automountServiceAccountToken`                      | bolean | true              | Automount service account token for the server service account.                              |
 
 
 ## Rqworker parameters
 
-Parameters specific to the `rqworkers` service of your Label Studio Enterprise deployment.
+Parameters are specific to the `rqworkers` service of your Label Studio Enterprise deployment.
+
+<i> Table 3: Rqworker parameters.</i>
 
 | Key                                                          | Type     | Default  | Description                                                                                 |
 |--------------------------------------------------------------|----------|----------|---------------------------------------------------------------------------------------------|
 | `rqworker.enabled`                                           | string   | true     | Enable rqworker pod                                                                         |
-| `rqworker.NameOverride`                                      | string   | ""       | String to partially override release template name                                          |
-| `rqworker.FullnameOverride`                                  | string   | ""       | String to fully override release template name                                              |
+| `rqworker.NameOverride`                                      | string   | true       | String to partially override release template name                                          |
+| `rqworker.FullnameOverride`                                  | string   | true       | String to fully override release template name                                              |
```

Comment:
not sure it should be true here too

Reply:
yes as per https://labelstud.io/guide/helm_values.html

Reply:
As per https://labelstud.io/guide/helm_values.html, it is set to true.